src/renderer/vt/VtSequences.cpp

```diff
@@ -434,6 +434,17 @@ using namespace Microsoft::Console::Render;
     return _Write("\x1b[?9001h");
 }
 
+// Method Description:
+// - Send a sequence to the connected terminal to switch to the alternate or main screen buffer.
+// Arguments:
+// - useAltBuffer: if true, switch to the malt buffer, otherwise to the main buffer.

```

Comment:
hehe `malt`

```suggestion
// - useAltBuffer: if true, switch to the alt buffer, otherwise to the main buffer.
```

(If comment updates are the only suggestions, I will admin merge without CI.)